dataflow/shell_command.go

```diff
@@ -20,7 +20,7 @@ import "os/exec"
 
 func DataflowShellCommand(fileName string, dataflowServerUrl string, skipSslValidation bool) *exec.Cmd {
 	cmd := exec.Command("java", "-jar", fileName, "--dataflow.uri="+dataflowServerUrl,
-		"--dataflow.credentials-provider-command=cf oauth-token")
+		"--dataflow.credentials-provider-command=cf oauth-token", "--dataflow.mode=skipper")
```

Comment:
@royclarkson yes, that confused me at first. It seems that this gets translated properly due to separation of each argument into its own parameter.

Reply:
The quotes are required because of how BASH parses parameters. Go doesn't try to be "smart" about parameter passing the same way BASH does so this is totally fine.